source/client-integration/create-client-configuration.rst

```diff
@@ -7,6 +7,62 @@ A client configuration includes all organization specific configuration and all
 
 ..  tabs::
 
+  ..  group-tab:: C#
+
+
+      .. NOTE::
+         SEID 2.0 enterprise certificates
+
+          If you have a SEID 2.0 enterprise certificate from Buypass, you need to use at least version `8.1.0 <https://github.com/digipost/signature-api-client-dotnet/releases/tag/8.1.0>`_ of the dotnet client library.
+          If this is not possible, or you have a SEID 2.0 enterprise certificate from Commfies, you need to disable validation of the enterprise certificate when configuring your client. The certificate will always be validated on our server, so it will not impact
```

Comment:
```suggestion
          If this is not possible, or you have a SEID 2.0 enterprise certificate from Commfides, you need to disable validation of the enterprise certificate when configuring your client. The certificate will always be validated on our server, so it will not impact
```